cool-core/src/test/java/com/nus/cool/core/model/CoolModelTest.java

```diff
@@ -52,6 +55,20 @@ public static void CsvLoaderTest() throws IOException {
         loader.load(cube, schemaFileName, dimFileName, dataFileName, cubeRepo);
     }
 

```

Comment:
we should do the exception test, but you should add such tests for each class.
For example, this CsvFileNonExistTest function should be created in the unit tests for the CsvLoader.
You can check PR #32 to learn how to create a new unit test file for a certain class.